plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -2150,27 +2156,26 @@ bool GSRendererHW::OI_JakGames(GSTexture* rt, GSTexture* ds, GSTextureCache::Sou
 	return true;
 }
 
-// OO (others output?) hacks: invalidate extra local memory after the draw call
-
-void GSRendererHW::OO_DBZBT2()
+bool GSRendererHW::OI_DBZBT2(GSTexture* rt, GSTexture* ds, GSTextureCache::Source* t)
 {
-	// palette readback (cannot detect yet, when fetching the texture later)
-
-	uint32 FBP = m_context->FRAME.Block();
-	uint32 TBP0 = m_context->TEX0.TBP0;
-
-	if(PRIM->TME && (FBP == 0x03c00 && TBP0 == 0x03c80 || FBP == 0x03ac0 && TBP0 == 0x03b40))
-	{
-		GIFRegBITBLTBUF BITBLTBUF;
-
-		BITBLTBUF.SBP = FBP;
-		BITBLTBUF.SBW = 1;
-		BITBLTBUF.SPSM = PSM_PSMCT32;
-
-		InvalidateLocalMem(BITBLTBUF, GSVector4i(0, 0, 64, 64));
-	}
+	// Sprite rendering
+	if (m_context->FRAME.PSM != PSM_PSMCT32)
+		return true;
+	if (PRIM->TME && m_context->TEX0.PSM != PSM_PSMCT32)
+		return true;
+	GSVector4i r = m_r;
+	if (!(r == GSVector4i(0, 0, 64, 64)).alltrue())  // Rendering region is 64x64
+		return true;
+	if (m_vt.m_eq.rgba != 0xffff || m_vt.m_eq.z != 0x1 || (PRIM->TME && !PRIM->FST && m_vt.m_eq.q != 0x1))  // No rasterization
+		return true;
+	if (m_context->DepthRead() || m_context->DepthWrite())  // No depth handling
+		return true;
+	SwSpriteRender();
+	return false; // Skip current draw
```

Comment:
Maybe add a empty row before return.
Bonus/optional, maybe move entire OI below Bigmutha OI instead?

Reply:
What's the point in moving it below Bigmutha OI ?

Reply:
Nothing major just alphabetical order for convenience.